sdk/tables/data-tables/swagger/README.md

```diff
@@ -62,3 +60,30 @@ directive:
     transform: >
       $["x-ms-client-name"] = "etag";
 ```
+
+### Make Access Policy not required
+
+```yaml
+directive:
+  from: swagger-document
+  where: $.definitions.SignedIdentifier
+  transform: >
+    $.required = ["Id"]
+```
+
+```yaml
+directive:
+  from: swagger-document
+  where: $.definitions.AccessPolicy
+  transform: >
+    delete $.required
+```
+
+```yaml
+directive:
+  from: swagger-document
+  where: $.definitions.AccessPolicy
+  transform: >
+    delete $.properties["Start"]["format"];
```

Comment:
This is removing the "date-time" format for these 2 properties. This is so that the mappers accept a string and we take care of serializing/deserializing dates to a truncated ISO string as the service expects. This is only for get and setAccessPolicy methods